cmake/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ option(onnxruntime_CROSS_COMPILING "Cross compiling onnx runtime" OFF)
 option(onnxruntime_BUILD_SERVER "Build ONNX Runtime Server" OFF)
 option(onnxruntime_USE_FULL_PROTOBUF "Use full protobuf" OFF)
 option(onnxruntime_DISABLE_CONTRIB_OPS "Disable contrib ops" OFF)
+option(onnxruntime_USE_WIN32_THREADPOOL "Use the default per-process Win32 threadpool" OFF)
 option(onnxruntime_USE_EIGEN_THREADPOOL "Use eigen threadpool. Otherwise OpenMP or a homemade one will be used" OFF)
```

Comment:
This description would make me think I should be enabling this. Could it maybe say if/when you'd actually want to enable it and why so it's clear this is probably something you want to leave turned off?

Reply:
Is this needed at all? I had tried to delete the Win32 thread pool code from MLAS in the first attempt at this.

Reply:
Not necessary.  Let me delete it.

Reply:
@tracysh Do you want mlas be able to run independently without onnxruntime, and the onnxruntime thread pool? If you keep the WIN32_THREADPOOL macro, it means mlas could be still multiple threaded without onnxruntime.

Reply:
Thanks for checking: in my standalone tests, I just use openmp for threading. The WIN32_THREADPOOL stuff was only brought back for ORT usage.

Reply:
Removed. Please check.